src/main/java/appeng/datagen/providers/models/BlockModelProvider.java

```diff
@@ -147,14 +147,27 @@ private void meChest() {
     }
 
     private void quartzGrowthAccelerator() {
-        var unpoweredModel = getExistingModel("block/growth_accelerator_off");
-        var poweredModel = getExistingModel("block/growth_accelerator_on");
+//        var unpoweredModel = getExistingModel("block/growth_accelerator_off");
+//        var poweredModel = getExistingModel("block/growth_accelerator_on");
```

Comment:
These two lines are old code an could be removed I guess.

Reply:
To be re-added at a future date